natlas-server/app/models/scope_item.py

```diff
@@ -28,8 +28,8 @@ class ScopeItem(db.Model, DictSerializable):  # type: ignore[misc, name-defined]
         lazy=True,
     )
     addr_family = db.Column(db.Integer)
-    start_addr = db.Column(db.VARBINARY(16))
-    stop_addr = db.Column(db.VARBINARY(16))
```

Comment:
These still produce a diff in the generation of a migration, but I haven't decided if I'm going to keep the old migrations around or if I'm going to delete them and start fresh with postgres-only.